README.md

```diff
@@ -98,17 +97,18 @@ When it terminates, the following command can be used to get access to
 the Kubernetes cluster that was brought up for testing (assuming that
 this step succeeded):
 
-    export KUBECONFIG="$(kind get kubeconfig-path --name="csi-prow")"
+`export KUBECONFIG="$(kind get kubeconfig-path --name="csi-prow")"`
 
 It is possible to control the execution via environment variables. See
 `prow.sh` for details. Particularly useful is testing against different
 Kubernetes releases:
 
-    CSI_PROW_KUBERNETES_VERSION=1.13.3 ./.prow.sh
-    CSI_PROW_KUBERNETES_VERSION=latest ./.prow.sh
+```shell
```

Comment:
Do we get better rendering with "console" instead of "shell"? "console" does syntax highlighting; I don't know whether "shell" does.

Reply:
Done.